common-npm-packages/utility-common/package.json

```diff
@@ -20,6 +20,7 @@
         "@types/node": "^16.11.39",
         "azure-pipelines-task-lib": "^4.11.0",
         "azure-pipelines-tool-lib": "^2.0.7",
+        "azure-devops-node-api": "14.0.1",
```

Comment:
Let's not add new dependencies to common utility library, I suggest to move this functionality to azure-arm-rest library